internal/http.go

```diff
@@ -36,41 +36,41 @@ func readResponse(r *http.Response) (*httpRes, error) {
 	return res, nil
 }
 
-func httpGet(client *http.Client, url string, headers map[string]string) (*httpRes, error) {
-	req, err := http.NewRequest("GET", url, nil)
+func httpDo(client *http.Client, method, url string, headers map[string]string, data interface{}) (*httpRes, error) {
+	var dataBytes []byte
+	if data != nil {
+		var err error
+		dataBytes, err = json.Marshal(data)
+		if err != nil {
+			return nil, err
+		}
+	}
+	req, err := http.NewRequest(method, url, bytes.NewBuffer(dataBytes))
 	if err != nil {
 		return nil, err
 	}
 	req.Header.Add("User-Agent", "fioconfig-client/2")
-	req.Close = true
-
+	req.Header.Add("Content-Type", "application/json")
```

Comment:
Will it work if `dataBytes`/`data` is `nil`?

Reply:
And the method is `GET`? I guess it will work, just sending not needed header around.